.github/workflows/wfc_lifecycle.yml

```diff
@@ -92,26 +92,30 @@ jobs:
     steps:
       - name: mark issues as stale
         run: |
-          before=`date -d '${{ inputs.daysBeforeStale }} days ago' --iso-8601`
-          echo "Getting all issues untouched since: $before"
-          issues=`gh issue list -S "-label:${{ inputs.staleLabel }} updated:<$before" -s open -R ${{ github.repository }} --json number -t '{{range .}}{{.number}} {{end}}'`
-          echo "Updating `echo $issues | wc -w` issues"
-          for issue in $issues; do
-            gh issue edit $issue --add-label ${{ inputs.staleLabel }} -R ${{ github.repository }}
-            gh issue comment $issue -R ${{ github.repository }} --body "$COMMENT"
-          done
+          if gh issue list --limit 1 --repo ${{ github.repository }} > /dev/null 2>&1; then
+            before=`date -d '${{ inputs.daysBeforeStale }} days ago' --iso-8601`
+            echo "Getting all issues untouched since: $before"
+            issues=`gh issue list -S "-label:${{ inputs.staleLabel }} updated:<$before" -s open -R ${{ github.repository }} --json number -t '{{range .}}{{.number}} {{end}}'`
+            echo "Updating `echo $issues | wc -w` issues"
+            for issue in $issues; do
+              gh issue edit $issue --add-label ${{ inputs.staleLabel }} -R ${{ github.repository }}
+              gh issue comment $issue -R ${{ github.repository }} --body "$COMMENT"
+            done
+          fi
         env:
           COMMENT: ${{ inputs.staleComment }}
       - name: close issues marked with label
         if: inputs.autocloseLabels
         run: |
-          issues=`gh issue list -S "label:${{ inputs.autocloseLabels }}" -s open -R ${{ github.repository }} --json number -t '{{range .}}{{.number}} {{end}}'`
-          echo "Auto closing issues `echo $issues | wc -w` issues"
-          for issue in $issues; do
-            gh issue edit $issue --remove-label ${{ inputs.openLabels }} -R ${{ github.repository }}
-            gh issue comment $issue -R ${{ github.repository }} --body "$COMMENT"
-            gh issue close $issue -R ${{ github.repository }}
-          done
+          if gh issue list --limit 1 --repo ${{ github.repository }} > /dev/null 2>&1; then
```

Comment:
Alternatively (sorry for the slow drip of comments) you can exit early with 

```
if ! gh issue list --limit 1 --repo ${{ github.repository }} > /dev/null 2>&1; then
  exit 0
fi
before=....
```

but either way is fine with me.

Reply:
Ah, yeah, I like this better. Agreed and changed.